index.html

```diff
@@ -77,19 +78,21 @@ <h1 class="logo">My Logo</h1>
                 or sign-ups required.
               </p>
             </div>
-            <div class="htmlButtons">
-              <button type="button">html</button>
-              <button type="button">css</button>
-              <button type="button">javascript</button>
-            </div>
+          
+              
+             <ul  class="htmlButtons" >
+              <li><button type="button">html</button></li>
+              <li><button type="button">css</button></li>
+              <li><button type="button">javascript</button></li>
+              </ul>
```

Comment:
- [ ] Kindly make sure you stick to the design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193132439-4b8617e7-0873-4ae7-a697-d38a0de0ba4a.png)
The Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193132746-436659a8-d804-4a8f-8dde-948cb6a025d7.png)